BookUniverseProject/BookUniverseProject/Models/LayoutModel.cs

```diff
@@ -0,0 +1,9 @@
+﻿using BookUniverse.Application.DTOs.CategoryDTOs;
+
+namespace BookUniverse.Web.Models
+{
+    public class LayoutModel
```

Comment:
Please change the name of LayoutModel in such a way that it is clear that the action is performed here with categories

Reply:
This LayoutModel is designed to manage not only categories but also folders and other possible menu items. So it should have more generic name, maybe MenuModel